readme.md

```diff
@@ -292,6 +292,10 @@ XO will handle the [@typescript-eslint/parser `project` option](https://github.c
 
 See [eslint-config-xo-flow#use-with-xo](https://github.com/xojs/eslint-config-xo-flow#use-with-xo)
 
+## GitHub Actions
+
+XO uses a different formatter when running in a GitHub Actions workflow to be able to get inline annotations. XO also disables warnings here.
```

Comment:
@fregante Do you know any page I could link to with more info about the inline annotations?

Reply:
Found these on their blog

https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-status-checks#checks
https://developer.github.com/changes/2019-09-06-more-check-annotations-shown-in-files-changed-tab/